src/arch/arm/table_walker.hh

```diff
@@ -264,7 +264,7 @@ class TableWalker : public ClockedObject
         bool
         secure(bool have_security, WalkerState *currState) const override
         {
-            if (have_security && currState->secureLookup) {
```

Comment:
Nit: this should be kept as secureLookup for consistency

Reply:
Done, somehow missed that in the last update.